x-pack/test/ftr_apis/security_and_spaces/config.ts

```diff
@@ -30,6 +30,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       serverArgs: [
         ...apiIntegrationConfig.get('kbnTestServer.serverArgs'),
         '--server.xsrf.disableProtection=true',
+        // disable internal API restriction. See https://github.com/elastic/kibana/issues/163654
+        '--server.restrictInternalApis=false',
```

Comment:
Set to `true` to enable restriction and run spaces api_integration tests. The tests should still pass.